### DISCUSSION TITLE
Easy way to make QTooltip trigger by QBtn click, not by hover?

### DISCUSSION THREAD

**Discussion by KingWebsites** [quasarframework/quasar#13098] [General - Components / Directives / etc] [answered]
I have a simple QBtn and QTooltip combo working well but the client want a click to trigger the tooltip, not a hover.

I can't find a simple example anywhere of this basic behaviour. Here's my existing standard 'hover' code:

```
<span v-if="mydata.tooltip">
  <q-btn flat round size="sm" padding="xs" :icon="outlinedInfo">
    <q-tooltip class="bg-primary q-pa-md text-body2 text-white" v-html="mydata.tooltip"/>
  </q-btn>
</span>
```

Edit: There will be multiple different tooltips on this page if that makes a difference.

**Comment by ontwikkelfabriek** [comment]
@KingWebsites 

Put a `ref` on the tooltip and then use the `@click` event of the q-button to call the `show` method of the q-tooltip( with the ref), see the q-tooltip Api doc for more info on the 'show' method:

https://quasar.dev/vue-components/tooltip#introduction

**Reply by KingWebsites** [reply]
I'm quite new to Vue so I don't quite understand what you're telling me.

1. Put a ref on the tooltip. I've used ref before but only for changing v-if variables. How do I put it on a tooltip?

2. For the @click event on the q-btn would this be ok? @click="toggle = !toggle"

3. Call the show method. Is this all I would have to do? show="toggle". How do I connect this to the ref?

I've added the following to my script component:

```
setup(props) {
  const toggle = ref(false);
  return {
    toggle
  }
}
```

**Comment by pdanpdan** [comment]
Use the model, Luke
https://github.com/quasarframework/quasar/blob/d2d96aa9c2cfcbef78b7f22d354ba8254c579b93/ui/dev/src/pages/components/tooltip.vue#L21

**Reply by KingWebsites** [reply]
I don't think it can work that way.

**Comment by MilosPaunovic** [comment, marked as answer]
As Dan suggested, simplest way is to use v-model:
[https://quasar.dev/vue-components/tooltip#example--toggle-through-v-model](https://quasar.dev/vue-components/tooltip#example--toggle-through-v-model)

**Reply by KingWebsites** [reply]
Thanks. From all the advice above I created this solution. The q-tooltip has to be outside the q-btn for it to work:

```
<span v-if="mydata.tooltip">
  <q-btn flat round size="sm" padding="xs" :icon="outlinedInfo" @click="showing = !showing"></q-btn>
  <q-tooltip class="bg-primary q-pa-md text-body2 text-white" v-model="showing" v-html="mydata.tooltip"/>
</span>
```

And in my setup script:
```
return {
  showing: ref(false),
};
```
